srml/democracy/src/lib.rs

```diff
@@ -156,7 +180,12 @@ decl_storage! {
 		/// The next referendum index that should be tallied.
 		pub NextTally get(next_tally) build(|_| 0 as ReferendumIndex): ReferendumIndex;
 		/// Information concerning any given referendum.
-		pub ReferendumInfoOf get(referendum_info): map ReferendumIndex => Option<(T::BlockNumber, T::Proposal, VoteThreshold)>;
+		pub ReferendumInfoOf get(referendum_info): map ReferendumIndex => Option<(ReferendumInfo<T::BlockNumber, T::Proposal>)>;
```

Comment:
shouldn't the return value also include the vote threshold and the delay block number? i.e. `Option<(ReferendumInfo<T::BlockNumber, T::Proposal, VoteThreshold, T::BlockNumber>)>`